### ISSUE TITLE
No plugin config files created !

### ISSUE THREAD

**Issue by 5UP3RTH30B4G** [Fr33styler/WorldRestore#1]
[01.10 19:19:27] [Server] [INFO] Actication of WorldRestore v1.01
[01.10 19:19:27] [Server] [INFO] Failed to delete the file with the plugin's directory name!

That what my console tell me

Version 1.12.2 Mohist
1.01

**Comment by Fr33styler** [comment]
That's a very weird error, i think you don't have write permission on that
location

**Comment by 5UP3RTH30B4G** [comment]
yeah that weird maybe that not the right name in the plugin ? cause I'm the owner of the server

**Comment by 5UP3RTH30B4G** [comment]
-- System Details --
Details:
Mohist version: 1.12.2-320
Operating System: Linux (amd64) version 4.15.0-151-generic
Java Version: 1.8.0_311, Oracle Corporation
Java VM version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
Memory: 1899359536 bytes (1811 MB) / 2625634304 bytes (2504 MB) up to 28631367680 bytes (27305 MB)
JVM flags: 0 total;
IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
Information from CraftBukkit:
    Running: Mohist version git-Mohist-1.12.2-320 (MC: 1.12.2) (Implementing API version 1.12.2-R0.1-SNAPSHOT) false

**Comment by Fr33styler** [comment]
The server configuration shouldn't be the problem
